.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "app/gui/.dev-env"]
-	path = app/gui/.dev-env
+[submodule "app/common/.dev-env"]
```

Comment:
You move .dev-env, but you didn’t touch `.env*` symlinks in `app/gui`. They should be removed as no longer valid.